src/requests.js

```diff
@@ -64,6 +64,44 @@ export function updateRequest(path, { data, permissions }, options = {}) {
 /**
  * @private
  */
+export function jsonPatchPermissionsRequest(
+  path,
+  permissions,
+  opType,
+  options = {}
+) {
+  const {
+    headers,
+    safe,
+    last_modified,
+  } = { ...requestDefaults, ...options };
+
+  const ops = [];
+
+  for (const type in permissions) {
```

Comment:
I'm wondering about adding a check for a valid permissions object here. Thoughts?